build/templates/job.template.yaml

```diff
@@ -18,13 +18,11 @@ spec:
         - effect: NoSchedule
           key: node-role.kubernetes.io/infra
           operator: Exists
-      backoffLimit: 6
```

Comment:
Do we want to remove this (just wondering if it was an intentional change or accidental)

Reply:
`backoffLimit` isn't a real field on `.spec.toleration.spec` for a job, plus I was seeing warnings about it when testing so I just removed it.